paper-badge.html

```diff
@@ -55,6 +61,8 @@
   <template>
     <style>
       :host {
+        --iron-icon-width: 12px;
```

Comment:
We tend to prefer being specific about which element the custom property targets, so something like: 
```css
iron-icon {
  --iron-icon-width: 12px;
  --iron-icon-height: 12px;
}
```